### PR TITLE
Update the dispatch screen (EXPOSUREAPP-3730)

### PR DESCRIPTION
**Pull request by Oliver-Zimmerman**
### Description

Dispatcher screen and text adjustments as provided in Figma designs. 

### Steps to reproduce
Go to dispatcher screen and admire the awesome new design... 

Closes JIRA ticket EXPOSUREAPP-3730

### CODE REVIEW

**Review by janetback — DISMISSED** [review]
UA approved.

**Review by chiljamgossow — DISMISSED** [review]
Looks fine to (tested on emulator)

**Review by janetback — DISMISSED** [review]
UA reviewed

File: Corona-Warn-App/src/main/res/layout/include_submission_dispatcher.xml [hunk 1]
```diff
@@ -54,6 +75,18 @@
                 app:layout_constraintStart_toStartOf="@+id/guideline_card_start"
                 app:layout_constraintTop_toBottomOf="@+id/submission_dispatcher_qr" />
 
+            <TextView
+                android:id="@+id/submission_dispatcher_already_positive_text"
```
Comment (d4rken) [inline Corona-Warn-App/src/main/res/layout/include_submission_dispatcher.xml:79]: The distance between the first title, and the first card, and the second title and the third card is inconsistent.

**Review by d4rken — CHANGES_REQUESTED** [review]
Small padding issue, besides that, looks good.

File: Corona-Warn-App/src/main/res/layout/include_submission_dispatcher.xml (hunk 1 above)
Comment (d4rken) [reply Corona-Warn-App/src/main/res/layout/include_submission_dispatcher.xml:79]: Screenshot from 2020-11-19 19-40-48

Comment (Oliver-Zimmerman) [reply Corona-Warn-App/src/main/res/layout/include_submission_dispatcher.xml:79]: @d4rken for some reason I am only seeing this now. I'll resolve it immediately. 👍

**Review by ralfgehrer — APPROVED** [review]
Looks good on Pixel (Android 10)

**Review by AndroidMedaGalaxy — APPROVED** [review]
LGTM

Approved \\( ﾟヮﾟ)/

**Comment by Diapolo** [comment]
> ### Description
> 
> Dispatcher screen and text adjustments as provided in Figma designs.
> ### Steps to reproduce
> 
> Go to dispatcher screen and admire the awesome new design...
> 
> Closes JIRA ticket EXPOSUREAPP-3730

Just an idea/suggestion for regular watchers of this repo: add some screenshots for changed UI elements in general :).

**Review by d4rken — APPROVED** [review]

**Pull request merged**